Sources/Box/Box.swift

```diff
@@ -0,0 +1,362 @@
+            buttonProvider = button
+            self.colorProvider = colorProvider
+        }
+
+        init(
+            style: Warp.BoxStyle,
+            title: String?,
+            subtitle: String,
+            linkProvider: ButtonConstructor?,
+            buttonProvider: ButtonConstructor?,
+            colorProvider: ColorProvider = Config.colorProvider
+        ) {
+            self.style = style
+            self.title = title
+            self.subtitle = subtitle
+            self.linkProvider = linkProvider
+            self.buttonProvider = buttonProvider
+            self.colorProvider = colorProvider
+        }
+
+        public var body: some View {
+            ZStack {
+                backgroundView
+
+                foregroundView
+                    .padding(.all, 24)
+            }
+            .cornerRadius(boxCornerRadius)
+            .accessibilityElement(children: .combine)
+            .modifier(
+                AccessibilityTraitBuilder(
+                    buttonProvider: buttonProvider,
+                    linkProvider: linkProvider
+                )
+            )
+            .modifier(AccessibilityButtonActionBuilder(buttonProvider: buttonProvider))
+        }
+
+        private var backgroundView: some View {
+            style.getBackgroundColor(from: colorProvider)
+                // View with rounded edges,
+                // and optional border.
+                .overlay(borderView)
+        }
+
+        @ViewBuilder
+        private var borderView: some View {
+            RoundedRectangle(cornerRadius: boxCornerRadius)
+                .stroke(style.getBorderColor(from: colorProvider), lineWidth: 2)
+        }
+
+        private var foregroundView: some View {
+            HStack(spacing: 8) {
+                toolTipIconView
+                    .accessibilityHidden(true)
+
+                informationView
+                    .padding(.leading, 16)
+
+                Spacer()
+            }
+        }
+
+        @ViewBuilder
+        private var toolTipIconView: some View {
+            if true {
+                VStack {
+                    Image(toolTipImageName, bundle: .module)
+                        .renderingMode(.template)
+                        .frame(width: 24, height: 24)
+                        .foregroundColor(colorProvider.labelIcon)
+                        .padding(.leading, 8)
+
+                    Spacer()
+                }
+            }
+        }
+
+        private var informationView: some View {
+            VStack(alignment: .leading, spacing: 8) {
+                titleView
+
+                subtitleView
+
+                linkView
+                    .padding(.vertical, 8)
+
+                buttonsView
+                    .padding(.top, 4)
+                    .padding(.bottom, 8)
+            }
+        }
+
+        @ViewBuilder
+        private var titleView: some View {
+            if let title {
+                Text(title)
```

Comment:
why not use our own Text(title, style: .title3)?

Reply:
Done, thanks.